package.json

```diff
@@ -3,7 +3,7 @@
   "repository": "gpuweb/types",
   "homepage": "https://github.com/gpuweb/types",
   "bugs": "https://github.com/gpuweb/types/issues",
-  "version": "0.1.38",
```

Comment:
nit: This shouldn't be incremented in this PR because it our publish script will increment it and we'll skip from 0.1.38 to 0.1.40 (which is not a big deal but slightly confusing)

Reply:
Done. I've reverted this change.